frontend/projects/upgrade/src/app/features/dashboard/feature-flags/pages/feature-flag-details-page/feature-flag-details-page-content/feature-flag-inclusions-section-card/feature-flag-inclusions-section-card.component.html

```diff
@@ -1,26 +1,37 @@
-<app-common-section-card>
-  <!-- header-left -->
-  <app-common-section-card-title-header
-    header-left
-    [title]="'feature-flags.details.inclusions.card.title.text' | translate"
-    [subtitle]="'feature-flags.details.inclusions.card.subtitle.text' | translate"
-    [tableRowCount]="tableRowCount$ | async"
-  ></app-common-section-card-title-header>
+<ng-container *ngIf="selectedFlag$ | async as flag">
+  <app-common-section-card>
+    <!-- header-left -->
+    <app-common-section-card-title-header
+      header-left
+      [title]="'feature-flags.details.inclusions.card.title.text' | translate"
+      [subtitle]="'feature-flags.details.inclusions.card.subtitle.text' | translate"
+      [tableRowCount]="rowCountWithInclude$ | async"
+    ></app-common-section-card-title-header>
 
-  <!-- header-right -->
-  <app-common-section-card-action-buttons
-    header-right
-    [showPrimaryButton]="true"
-    [primaryButtonText]="'feature-flags.details.add-include-list.button.text' | translate"
-    [showMenuButton]="true"
-    [menuButtonItems]="menuButtonItems"
-    [isSectionCardExpanded]="isSectionCardExpanded"
-    (primaryButtonClick)="addIncludeListClicked()"
-    (menuButtonItemClick)="onMenuButtonItemClick($event)"
-    (sectionCardExpandChange)="onSectionCardExpandChange($event)"
-  >
-  </app-common-section-card-action-buttons>
+    <!-- header-right -->
+    <app-common-section-card-action-buttons
+      header-right
+      [showSlideToggle]="true"
+      [slideToggleText]="'feature-flags.details.inclusions.include-all.toggle.text' | translate"
+      [isEnableToggleChecked]="flag.filterMode === FILTER_MODE.INCLUDE_ALL"
+      [showPrimaryButton]="true"
+      [primaryButtonText]="'feature-flags.details.add-include-list.button.text' | translate"
+      [showMenuButton]="true"
+      [menuButtonItems]="menuButtonItems"
+      [isSectionCardExpanded]="isSectionCardExpanded"
+      [primaryActionBtnDisabled]="flag.filterMode === FILTER_MODE.INCLUDE_ALL"
+      (primaryButtonClick)="addIncludeListClicked()"
+      (slideToggleChange)="onSlideToggleChange($event, flag.id)"
+      (menuButtonItemClick)="onMenuButtonItemClick($event)"
+      (sectionCardExpandChange)="onSectionCardExpandChange($event)"
+    >
+    </app-common-section-card-action-buttons>
 
-  <!-- content -->
-  <app-feature-flag-inclusions-table content *ngIf="isSectionCardExpanded"></app-feature-flag-inclusions-table>
-</app-common-section-card>
+    <!-- content -->
+    <app-feature-flag-inclusions-table
+      content
+      *ngIf="isSectionCardExpanded && flag.filterMode !== FILTER_MODE.INCLUDE_ALL"
+      class="full-width"
```

Comment:
I think the more proper way to do this is to update the `isSectionCardExpanded` status when the `filterMode` changes.